core/fetcher/fetcher.go

```diff
@@ -149,9 +149,10 @@ func (f *Fetcher) fetchProposerData(ctx context.Context, slot int64, defSet core
 
 		randao := randaoData.Signature().ToETH2()
 
-		// TODO(dhruv): what to do with graffiti?
-		// passing empty graffiti since it is not required in API
+		// TODO(dhruv): replace hardcoded graffiti with the one from cluster-lock.json
 		var graffiti [32]byte
+		s := []byte("Obol Distributed Validator")
```

Comment:
Proposal. Checkout the default grafitti for  all of the clients running here https://codebeautify.org/string-hex-converter

Would it be long term  more  value  add to make the string  `charon/version-sha` so we can see what versions of old clusters are still out there and live as they produce blocks?